classes/submission/PKPSubmission.php

```diff
@@ -289,6 +291,59 @@ public function getUIDisplayString()
     {
         return __('plugins.importexport.submission.cli.display', ['submissionId' => $this->getId(), 'submissionTitle' => $this->getCurrentPublication()->getLocalizedTitle()]);
     }
+
+    /**
+     * Return a collection of all existing JAV Stages 
+     * for the submission's publications
+     *
+     * @return Collection<JavStageAndNumbering>
+     */
+    public function getExistingJavStagesByPublications(): Collection
+    {
+        $publications = $this->getData('publications');
+        if ($publications->isEmpty()) {
```

Comment:
Aren't these 3 lines redundant? If the collection was empty, the result of this operation will be an empty collection anyway.